app/build.gradle

```diff
@@ -106,7 +106,7 @@ android {
             applicationIdSuffix ".prerelease"
             resValue "string", "app_name", "Tree Tracker"
             buildConfigField "Boolean", "GPS_ACCURACY", "true"
-            buildConfigField 'String', 'BASE_URL', '"https://treetracker.org/api/mobile/"'
+            buildConfigField 'String', 'BASE_URL', '"http://67.205.142.69/api/mobile/"'
```

Comment:
Why was this changed?

Reply:
Oh snap, thanks, that was something I while testing out the SSL on treetracker.org.  build.gradle gets auto-committed with its version bump when I push a new build, so this slipped in.